test/unix/main.ml

```diff
@@ -900,7 +906,7 @@ module Throttle = struct
 end
 
 let () =
-  Common.report ();
+  (* Common.report (); *)
```

Comment:
```suggestion
  Common.report ();
```

Reply:
is there any reason that this is commented out?